.github/workflows/quality.yml

```diff
@@ -0,0 +1,54 @@
+name: Code Quality
+on:
+   schedule: # weekly
+     - cron: '0 0 * * 0'
+   push:
+      branches:
+        - main
+        - test
```

Comment:
Have we started using this branch scheme yet?

Reply:
yeah.  push:main is essentiall merge PR.   push:test is a branch I sometimes create just to trigger tests on my fork to avoid messing with main.  (That said,  I'm inconsistent.)